domains/cinego.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "email": "virajjadhav512@gmail.com",
+    "github": "DeveloperViraj"
+  },
+  "record": {
+    "CNAME": "https://cinego-chi.vercel.app/"
```

CNAME doesnt support any trailing slashes (/)
```suggestion
    "CNAME": "cinego-chi.vercel.app"
```